src/hooks/useDeployment.ts

```diff
@@ -72,8 +72,8 @@ export function useDeployment() {
 
         // 8. deploy VkRegistry contract
         setDeployStatus(8);
-        // TODO: should be updated to read filepath and name from user input
-        const vks = require("./vk_10-2-1-2_test.0.json");
+
+        const vks = await fetch(config.vkeyFilePath).then((res) => res.json());
```

Comment:
I assume in another PR we'll allow users to not download every time but actually just copy paste/import from file the vkeys? as the download could easily take couple of minutes

Reply:
so this actually allows them to put existing vkey file in their `public/` folder to avoid downloading it everytime.
But of course in the future we could fetch from the maci-coordinator service.